lib/Service/Appointments/BookingCalendarWriter.php

```diff
@@ -168,7 +173,54 @@ public function write(AppointmentConfig $config, DateTimeImmutable $start, strin
 		try {
 			$calendar->createFromString($filename . '.ics', $vcalendar->serialize());
 		} catch (CalendarException $e) {
-			throw new RuntimeException('Could not write to calendar', 0, $e);
+			throw new RuntimeException('Could not write event  for appt config id ' . $config->getId(). ' to calendar', 0, $e);
+		}
+
+		if ($config->getPreparationDuration() !== 0) {
+			$string = $this->l10n->t('Preparation time for %s', [$config->getName()]);
```

Comment:
```suggestion
			$string = $this->l10n->t('Prepare for %s', [$config->getName()]);
```

in the hopes of making more text readable on narrow calendar cards